geode-redis/src/distributedTest/java/org/apache/geode/redis/RedisDistDUnitTest.java

```diff
@@ -209,18 +209,6 @@ public void run() throws InterruptedException {
             } else {
               jedis.del(hKey);
             }
-          } else if (n == 1) {
```

Comment:
Can we get rid of the @Ignore on this test now that we're only testing hashes and sets?

Reply:
done